### PR TITLE
cert-manager: fast-forward to upstream 193c5e2

### PR DESCRIPTION
**Pull request by jessynt**
#### What this PR does / why we need it:

* Bump to [v0.7.0](https://github.com/jetstack/cert-manager/releases/tag/v0.7.0)

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/cert-manager/README.md [hunk 1]
```diff
@@ -13,23 +13,25 @@ to renew certificates at an appropriate time before expiry.
...
+## IMPORTANT: if the cert-manager namespace **already exists**, you MUST ensure
+## it has an additional label on it in order for the deployment to succeed
+$ kubectl label namespace cert-manager certmanager.k8s.io/disable-validation="true"
+
+## Add the Jetstack Helm repository
+$ helm repo add jetstack https://charts.jetstack.io
 
 ## Install the cert-manager helm chart
-$ helm install --name my-release stable/cert-manager
+$ helm install --name my-release --namespace cert-manager jetstack/cert-manager
```
Comment (paulczar) [inline stable/cert-manager/README.md:34]: shouldn't this still be `stable/cert-manager` ?

**Review by paulczar — COMMENTED** [review]
I notice you have the jetstack/cert-manager repo up on hub.helm.sh ...   would it make sense to deprecate the version here in favor of that one ?

**Comment by jessynt** [comment]
> I notice you have the jetstack/cert-manager repo up on hub.helm.sh ... would it make sense to deprecate the version here in favor of that one ?

Thanks for the info @paulczar . I'll close this PR.

**Pull request closed without merging**